networking/src/ModelHelper.ts

```diff
@@ -129,25 +130,62 @@ export abstract class ModelHelper {
         const answers = this.getBasicTeamMemberAnswersToQuestionId(team, question.id)
         if (isNullOrUndefined(answers) ||
             answers.length === 0) {
-            return 0
+                console.log("answer length is 0");
+                return 0;
         }
 
         const correctAnswer = this.getCorrectAnswer(question)
         const currentQuestion = gameSession?.questions[gameSession?.currentQuestionIndex ?? 0]
         let submittedTrickAnswer = answers.find(answer => (!this.isAnswerFromPhaseOne(answer)) && answer?.questionId === currentQuestion.id)
 
         if (submittedTrickAnswer){
-          return ModelHelper.calculateBasicModeWrongAnswerScore(gameSession, submittedTrickAnswer.text ?? '', currentQuestion.id)
+            console.log("HERE");
+            return ModelHelper.calculateBasicModeWrongAnswerScore(gameSession, submittedTrickAnswer.text ?? '', currentQuestion.id)
         } else {
-            if (!isShortAnswerEnabled && answers.find(answer => (this.isAnswerFromPhaseOne(answer)) && answer?.text === correctAnswer?.text && answer?.questionId === currentQuestion.id && gameSession?.currentState === GameSessionState.PHASE_1_RESULTS)){
-                return this.correctAnswerScore
+            // loads of logging to see whats going on
+            console.log("isShortAnswerEnabled:", isShortAnswerEnabled);
+            console.log("answers:", answers);
+            console.log("correctAnswer:", correctAnswer);
+            console.log("currentQuestion:", currentQuestion);
+            console.log("gameSession.currentState:", gameSession?.currentState);
+    
+            const answerFound = answers.find(answer => {
+                const fromPhaseOne = this.isAnswerFromPhaseOne(answer);
+                const textMatch = answer?.text === correctAnswer?.text;
+                const questionIdMatch = answer?.questionId === currentQuestion.id;
+                // changed this...
```

Comment:
restate comment for more precision

Reply:
got rid of the whole answerFound situation. it was just used for debugging